ruleguard_rules.go

```diff
@@ -0,0 +1,25 @@
+//+build never
+
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package gorules
```

Comment:
isn't this package bogus for the root of `k8s.io/kubernetes`?

Reply:
there's build tags at the root to make it not build but this is kinda an ugly hack IMO.

also placing things in the repo root makes it ~impossible to delegate ownership at the moment.

Reply:
we don't really have any code there and this is `+build never`  -  what should it say?